AmplifyPlugins/API/Tests/APIHostApp/GraphQLAPIStressTests/README.md

```diff
@@ -0,0 +1,246 @@
+## GraphQL API Stress Tests
+
+The following steps demonstrate how to set up a GraphQL endpoint with AppSync. The auth configured will be API key.
+
+The steps tested with Amplify CLI version 9.1.0. When running a later CLI version, you may also want to update the generated model files that are placed in the Models folder. 
+
+Run `amplify codegen models` from your amplify app, compare and replace if needed the ones in the Models folder.
+
+
+### Set-up
+
+1. `amplify init`. Update `cli.json` values to use Transformer V1.
+
+- `"useexperimentalpipelinedtransformer": false`
+- `"transformerversion": 1`
```

Comment:
we shouldn't use TransformerV1 as we would like to deprecate this usage, so we should especially dogfood the latest transformer process in our integration tests to make sure we're using the same backend as new developers would

Reply:
Updated code